zebra-network/src/isolated/tor.rs

```diff
@@ -0,0 +1,91 @@
+//! Uses tor to create isolated and anonymised connections to specific peers.
+
+use std::sync::{Arc, Mutex};
+
+use arti_client::{TorAddr, TorClient, TorClientConfig};
+use tor_rtcompat::tokio::TokioRuntimeHandle;
+use tower::util::BoxService;
+
+use zebra_chain::parameters::Network;
+
+use crate::{connect_isolated, BoxError, Request, Response};
+
+#[cfg(test)]
+mod tests;
+
+lazy_static::lazy_static! {
+    /// The shared isolated [`TorClient`] instance.
+    ///
+    /// TODO: turn this into a tower service that takes a hostname, and returns an `arti_client::DataStream`
```

Comment:
Nit: documentation link.

```suggestion
    /// TODO: turn this into a tower service that takes a hostname, and returns an [`arti_client::DataStream`]
```